### DISCUSSION TITLE
Add a 'title' directive

### DISCUSSION THREAD

**Discussion by alexsmirnovdotcom** [alpinejs/alpine#885] [3. Feature ideas]
Hi! Thanks for awesome framework!
I working with Alpine and Livewire, and sometimes need to set "title" attribute in html tag, but now a can't do it.
Maybe i bad looking, but not find how to do it.
Maybe you can add `x-title` directive, or there are other ways to set this attribute with Alpine?
Thanks!

**Comment by SimoTod** [comment]
x-bind:title or :title for short
See https://github.com/alpinejs/alpine/#x-bind
